frontend/src/components/Card.tsx

```diff
@@ -0,0 +1,21 @@
+import { NobelRes } from "../types/type";
+
+export const Card = ({ nobel }: { nobel: NobelRes }) => {
```

Comment:
This looks very impressive code-wise, also interesting that you chose to work with typescript!